ticdc/ticdc-open-protocol.md

```diff
@@ -287,3 +344,41 @@ The column and DDL type codes are encodings defined by TiCDC Open Protocol. `Col
 | Create Sequence                   | 34 |
 | Alter Sequence                    | 35 |
 | Drop Sequence                     | 36 |
+
+## Bit flags of columns
+
+The bit flags represent specific attributes of columns.
+
+| Bit | Value | Name | Description |
+| :-- | :- | :- | :- |
+| 1   | 0x01 | BinaryFlag          | Whether the column is a binary-encoded column. |
+| 2   | 0x02 | HandleKeyFlag       | Whether the column is a Handle index column. |
+| 3   | 0x04 | GeneratedColumnFlag | Whether the column is a generated column.     |
+| 4   | 0x08 | PrimaryKeyFlag      | Whether the column is a primary key column.      |
+| 5   | 0x10 | UniqueKeyFlag       | Whether the column is a unique index column.  |
+| 6   | 0x20 | MultipleKeyFlag     | Whether the column is a composite index column.   |
+| 7   | 0x40 | NullableFlag        | Whether the column is a nullable column.       |
+| 8   | 0x80 | UnsignedFlag        | Whether the column is an unsigned column.     |
+
+Example:
+
+If the value of a column flag is `85`, the column is a nullable column, a unique index column, a generated column, and a binary-encoded column.
+
+```
+85 == 0b_101_0101
+   == NullableFlag | UniqueKeyFlag | GeneratedColumnFlag | BinaryFlag
+```
+
+If the value of a column is `46`, the column is a composite index column, a primary key column, a generated column, and a Handle key column.
+
+```
+46 == 0b_010_1110
+   == MultipleKeyFlag | PrimaryKeyFlag | GeneratedColumnFlag | HandleKeyFlag
+```
+
+> **Note:**
+>
+> + This feature is still experimental. Do **NOT** use it in the production environment.
+> + Currently, for a known composite index column, the `UniqueKeyFlag` value of this column is incorrectly output.
```

Comment:
Seems this bug has been fixed, I will confirm it later.

Reply:
I have confirmed this bug has been fixed, we can remove L382. Besides the Chinese document also needs to update.

Reply:
```suggestion
```